.github/workflows/test_notebooks.yml

```diff
@@ -5,15 +5,19 @@ on:
   push:
     branches: [ develop, stable, nbtests ]
     paths-ignore:
-      - '**/*.md' # ignore markdown files
-      - '**/*.rst' # ignore restructured text files
+      - 'DEA_Sandbox.ipynb' # ignore landing page
+      - 'DEA_notebooks_template.ipynb' # ignore template
+      - '*.md' # ignore all markdown files
+      - '*.rst' # ignore all restructured text files
       - '.github/**' # ignore anything in .github folder
       - '!.github/workflows/test_notebooks.yml' # except test_notebooks.yml
   pull_request:
     branches: [ develop, stable ]
     paths-ignore:
-      - '**/*.md'
-      - '**/*.rst'
+      - 'DEA_Sandbox.ipynb'
```

Comment:
Are you sure this should be ignored? It is a Notebook file that we need to check renders properly on the Sandbox.

Reply:
It doesn't actually include any code - it's all entirely pre-rendered markdown cells so there's not anything for `pytest` to actually run to test for a pass/fail.